### PR TITLE
Update Slack welcome message

### PR DESCRIPTION
**Pull request by Relequestual**
Reduce the size of the greeting message and remove the channel listing to another file.
(Please review only 2nd commit. First was a file move and rename.)

Note: This will need to be manually copied to Slack Workflows. One of the reasosn for taking a look at this again is that GreetBot is closing down.

Slack Workflows doesn't SEEM to accept markdown, so the links will have to be manually converted. Slack offer no function to programatically create or update workflows, sadly.

### CODE REVIEW

File: slack/greeting [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Hey {{Joiner}}, Welcome to the JSON Schema Slack Server!
+
+Whatever reason you chose to join us, we want you to know, you're welcome!
```
Comment (gregsdennis) [inline slack/greeting:3]: "Whatever the reason"?  Maybe that's just an American thing?

Also "you're welcome" might be misinterpreted as a response to "thank you."  Maybe, "we're glad you're here" or "we're glad to have you" or something.

File: slack/greeting [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+We have a formal Code of Conduct which we [invite you to review](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md). It is short, and reading it is respectful to the community you've chosen to enter.
```
Comment (gregsdennis) [inline slack/greeting:5]: Move the link to "Code of Conduct".  That's the thing that the link leads to.

File: slack/greeting [hunk 3]
```diff
@@ -0,0 +1,19 @@
+Hey {{Joiner}}, Welcome to the JSON Schema Slack Server!
+
+Whatever the reason you chose to join us, we want you to know, we're glad you've here.
```
Comment (jdesrosiers) [inline slack/greeting:3]:
```suggestion
Whatever the reason you chose to join us, we want you to know, we're glad you're here.
```

File: slack/other channels [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Here are some of our other channels and why you might want to join them.
+
+#implementers - Those implementing the specification and questions about implementation details
+#specification - Potential changes to the JSON Schema specification
+#tests - Development of the official test suite
+#community - Development of the JSON Schema community and organisation
```
Comment (jdesrosiers) [inline slack/other channels:6]: Should we standardize on American spelling or British spelling?

```suggestion
#community - Development of the JSON Schema community and organization
```

File: slack/other channels [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+#in-the-wild - Posting of any references to uses of JSON Schema by others for tracking
```
Comment (jdesrosiers) [inline slack/other channels:7]: This sentence feels awkward. I don't have a specific suggestion, but it might be worth having another look.

**Review by jdesrosiers — CHANGES_REQUESTED** [review]
I don't have much of an opinion on the content, but I did find a typo and a couple other places that could use another look.

File: slack/other channels (hunk 5 above)
Comment (gregsdennis) [reply slack/other channels:7]: "Find out and advertise where JSON Schema is used."  ?

File: slack/greeting (hunk 3 above)
Comment (Relequestual) [reply slack/greeting]: 🤦‍♂️ Thank you!

File: slack/other channels (hunk 4 above)
Comment (Relequestual) [reply slack/other channels:6]: Reluctantly, I agree.
I think for all formal content, yes.
I'm not as strict with the blog articles, but maybe we should be?

**Review by gregsdennis — APPROVED** [review]

**Review by jdesrosiers — APPROVED** [review]

File: slack/other channels (hunk 4 above)
Comment (karenetheridge) [reply slack/other channels]: > Should we standardize on American spelling or British spelling?

I feel very left out by this comment :D

File: slack/other channels (hunk 5 above)
Comment (karenetheridge) [reply slack/other channels]: "References to observed uses of JSON Schemas in other applications or projects"

**Pull request merged**